src/shared/lib/StoreUtils.ts

```diff
@@ -1856,74 +1850,78 @@ export function getOncoKbOncogenic(response: IndicatorQueryResp): string {
     }
 }
 
-export function evaluateDiscreteCNAPutativeDriverInfo(
-    cnaDatum: CustomDriverNumericGeneMolecularData,
-    oncoKbDatum: IndicatorQueryResp | undefined | null | false,
-    customDriverAnnotationsActive: boolean,
-    customDriverTierSelection: ObservableMap<string, boolean> | undefined
-) {
-    const oncoKb = oncoKbDatum ? getOncoKbOncogenic(oncoKbDatum) : '';
+type HotSpotInfo = {
```

Comment:
we should put these types into a model file in /src/shared/model folder.

Reply:
Fixed in [860b4a8](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/860b4a806eabd2b81d208148dfdcacd5db0b6661)